### PR TITLE
Add freesurfer_queries_bsf2016.qry

### PR DESCRIPTION
**Pull request by cbedetti**
Queries from:
"The white matter query language: a novel approach for describing human white matter anatomy"
Brain Structure and Function 2016

### CODE REVIEW

File: tract_querier/data/freesurfer_queries_bsf2016.qry [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+premotor.side |= (
+    rostralanteriorcingulate.side or
+    caudalanteriorcingulate.side or
+    caudalmiddlefrontal.side or
+    parsopercularis.side
+)
+
+striatum.side |= putamen.side or pallidum.side or caudate.side
+
+# Not used in BSF 2016
```
Comment (cbedetti) [inline tract_querier/data/freesurfer_queries_bsf2016.qry]: Next commented queries are not used in BSF2016.
I kept them as references.

**Comment by arnaudbore** [comment]
Are we sure about these queries before accepting the pull request ?
 
'motor.side' should be 'precentral.side'
I think that 'prefrontal.side' is overestimated because it includes frontal as well.

I'm not a neuro-anatomist but we need to be sure about these queries.

@demianw can you take a closer look on these queries or we can find somebody to take sometime as well.

Thank again for the WMQL.

**Comment by demianw** [comment]
Thanks flor checking @arnaudbore . I think those queries are very close to those that neuroanatomist Nikos Makris and I developed. The prefrontal cortex is included composed of sections that are on the frontal lobe in the Desikan-Killiany atlas. I will work on this PR next week when I'm back from my travels.

**Comment by demianw** [comment]
@arnaudbore is right the line should be
`motor.side |= pre central.side`
it was my mistake when I gave the query to @cbedetti

**Comment by demianw** [comment]
Hi @arnaudbore @cbedetti 
OK, I double checked some of the issues with Nikos Makris and cross-referenced others. And there are some minor adjustments to what I initially gave to @cbedetti . 
We must take in account that these labels do not map exactly to Desikan-Killiany parcellation (hence the need for the posterior_of) 

```
motor.side |= precentral.side

superior_parietal_lobule.side |= (
      precuneus.side or
      superiorparietal.side
 )

posterior_parietal_lobule.side |= (
    supramarginal.side or
    inferioriorparietal.side 
)

premotor.side |= (
      rostralanteriorcingulate.side or
      parsopercularis.side or
      (superiorfrontal.side and posterior_of(caudalmiddlefrontal.side)) 
 )

prefrontal.side |= (
      parstriangularis.side or
      parsorbitalis.side or
      superiorfrontal.side or
      rostralmiddlefrontal.side or
      frontalpole.side or
      lateralorbitofrontal.side
  )
```

Do you agree with these @arnaudbore? If so, @cbedetti would you like to make the changes or should I?
On a different point, I think we should eliminate the commented queries.

Thanks!

**Comment by cbedetti** [comment]
Hi,
I added the corrections to the pull request.
Thank you @demianw.

**Comment by demianw** [comment]
@cbedetti great! Did you check that they work well or that they have the expected result?

**Comment by arnaudbore** [comment]
Hi,
I checked with a PhD student and we still have some concern about the definition of the premotor.
Using this [atlas](http://thebrain.mcgill.ca/flash/a/a_06/a_06_cr/a_06_cr_mou/a_06_cr_mou.html) we would like to suggest:

premotor.side |= caudalmiddlefrontal.side or parsopercularis

**Comment by cbedetti** [comment]
Hi,

I understand some changes are beyond the bsf2016 article. When I checked the images with @arnaudbore and the PhD student, we decide to submit these changes.

- the premotor definition of @arnaudbore seems better (see `cc3` on the right)
- `not in hemisphere.opposite` was added for the cortico-spinal tract
- some exclusion regions was added for the uncinate fascicle

2016-08-26_comp

**Comment by demianw** [comment]
Hi and thanks for this.

I agree with you on the definition of the premotor cortex. The posterior section superior frontal gyrus is problematic when defined through the Desikan atlas.

Regarding the exclusion queries, I'm parted, in the BSF paper we avoided their use in anatomical definitions. This is mainly because they are often used to deal with false positive tracts which differ on different tractography algorithms.

I propose this: let's keep your version of the PMC the you can generate a new query file which imports this one and adds the exclusions to these specific tracts. Maybe add a suffix to the current file name with the name of the algorithm or software you used to generate the tracts?

What say you?

**Comment by cbedetti** [comment]
Hi,
Seems reasonable to do it this way.
I made the changes and the two `.qry` are tested.
I added comment to explain `freesurfer_queries_bsf2016_toad.qry`.
Let me know what you think.

**Comment by demianw** [comment]
@cbedetti 
This is great, thanks!
I am for merging, just one more thing: would you like to add yourself as the author or co-author in the files?

If not, I'll just merge

Thanks for all this work!

**Comment by cbedetti** [comment]
Hi,
I added the informations.
Thank you for the opportunity.

**Comment by demianw** [comment]
@cbedetti great! I'm merging! And then I'm adding myself as co-author of  freesurfer_queries_bsf2016.qry as I think I did a part of that work too

Thanks for this work

**Pull request merged**